index.js

```diff
@@ -1,17 +1,56 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
+  add(item) {
+    this.items.push(item);
+    let sortedItems = this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
+    return sortedItems;
+  }
 
-  add(item) {}
+  get(pos) {
+    if (this.items[pos] !== undefined) {
+      return this.items[pos];
+    } else {
+      throw new Error("OutOfBounds");
+    }
+  }
 
-  get(pos) {}
+  max() {
+    if (this.items.length === 0) {
+      throw new Error("EmptySortedList");
+    }
+    return Math.max(...this.items);
+  }
 
-  max() {}
+  min() {
+    if (this.items.length === 0) {
+      throw new Error("EmptySortedList");
+    }
+    return Math.min(...this.items);
+  }
 
-  min() {}
+  sum() {
+    if (this.items.length === 0) {
+      return 0;
+    }
+    let sum = this.items.reduce(function (acc, el) {
+      return acc + el;
+    }, 0);
```

Comment:
Since we'll be using ES6 syntax a lot from now on, the reduce method could also be written like this: `.reduce((acc, el) => acc + el, 0);`